### ISSUE TITLE
Color picker not fully accessible

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#1659]
### Description

The content of the color picker used in various locations in the Pressbooks interface (inherited from WordPress) is not communicated to assistive technologies. The selector within the gradient is communicated as “link”, and colors (black, white, red, etc.) are announced as group. Can be fixed by adding an ARIA label to each color so that it says something other than 'group' (i.e. the color's name).

**Comment by SteelWagstaff** [comment]
Reported upstream: https://github.com/Automattic/Iris/issues/69

**Comment by mcgratay** [comment]
Resolved.

**Comment by mcgratay** [comment]
Resolved.
